frontend/src/components/communication/MessageContent.js

```diff
@@ -61,6 +61,8 @@ export default function MessageContent({ content, renderYoutubeVideos }) {
     var ID = "";
     url = url.replace(/(>|<)/gi, "").split(/(vi\/|v=|\/v\/|youtu\.be\/|\/embed\/)/);
     if (url[2] !== undefined) {
+      // TODO: confirm this escape character is required
+      // eslint-disable-next-line no-useless-escape
       ID = url[2].split(/[^0-9a-z_\-]/i);
```

Comment:
Left a TODO for this as I wanted to confirm whether it was needed.

Reply:
I am not really sure, to be honest, so I'd just leave it in to be safe unless you're sure that it can be removed.